source/docs/05-mensural.xml

```diff
@@ -92,8 +92,8 @@
-            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in which the symbol appears. The general context is given by one of the 16 mensural <hi rend="italic">species</hi> provide four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Depending on the context, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly desirable.</p>
-            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.</p>
+            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by the position and the context in which the symbol appears. The general context is given by one of the 16 mensural <hi rend="italic">species</hi> provided by the four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Then, depending on the specific context where the note is positioned, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly desirable.</p>
+            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric ratios or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.</p>
```

Comment:
```suggestion
            <p>Another difference is the use of proportions that are indicated by numeric ratios or by specific mensuration signs. The proportions indicate that the durations have to be modified and they may be combined. Proportions and mensuration signs were eventually simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.</p>
```

Reply:
"The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module"

This sentence seems redundant. Where else would they be defined?